spec/plaindatetime.html

```diff
@@ -879,8 +879,10 @@ <h1>InterpretTemporalDateTimeFields ( _calendar_, _fields_, _options_ )</h1>
       </p>
       <emu-alg>
         1. Let _timeResult_ be ? ToTemporalTimeRecord(_fields_).
-        1. Let _temporalDate_ be ? DateFromFields(_calendar_, _fields_, _options_).
         1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+        1. Let _optionsCopy_ be ! OrdinaryObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_optionsCopy_, *"overflow"*, _overflow_).
+        1. Let _temporalDate_ be ? DateFromFields(_calendar_, _fields_, _optionsCopy_).
```

Comment:
This seems like it drops all the options - was this meant to create with a proto of options rather than null?

Reply:
Good point, that makes good sense for this call site since `_overflow_` is the only point we don't want to observe modifications. Do you agree for the other PlainYearMonth call sites it makes sense to have a null proto and only pass overflow?

Reply:
I’m not sure - i assume that if it’s only passing specific options across, then those are the only options that are listened to - but it seems like user code would want maximum information available, meaning they’d want all options to be copied over.

Reply:
Ah, I've been away a little while, I'd forgotten about ToPrimitive. I'll create a proper copy :)

Reply:
I think for this case it'd be better to just switch the order of steps 2 and 3, reading the overflow property before handing the object over to user code.

Reply:
True, but it’d still be better if a mutable object wasn’t persisted through every call :-)

Reply:
only one call into user code in this case, though, so nbd! I'll go with the order swap.